src/core/Tw2VariableStore.js

```diff
@@ -34,41 +34,49 @@ Tw2VariableStore.prototype.RegisterType = function(name, type)
 };
 
 /**
- * Registers a variable without a type
- * @param {string} name
- * @param {string|number|Float32Array|vec3|mat4} value
- * @returns {Tw2Parameter}
- * @constructor
+ * Gets A Tw2 parameter constructor from a supplied value
+ * @param {Number|String|Array.<Number>|Float32Array} value
+ * @returns {null|Function}
  */
-Tw2VariableStore.prototype.RegisterVariable = function(name, value)
+Tw2VariableStore.GetTw2ParameterType = function (value)
 {
-    if (value.constructor == (new glMatrixArrayType()).constructor)
+    if (value.constructor === (new glMatrixArrayType()).constructor || Object.prototype.toString.apply(value) === '[object Array]')
```

Comment:
Yes it is better, apparently 5x faster than the to string method which I have always used, so thank you for that!